lib/src/widget/eh_gallery_torrents_dialog.dart

```diff
@@ -118,7 +118,7 @@ class _TorrentList extends StatelessWidget {
                   (_) => toast('hasCopiedToClipboard'.tr),
                 ),
               ),
-            ) as Widget,
```

Comment:
Here we must explicitly force the type to `Widget` due to the method `joinNewElement` below, otherwise i can't compile successfully. We should keep `as Widget`.

Reply:
i see i'll add it back

Reply:
also i think i would be better to explicitly cast it in the map() method